src/styles/components/_footer.scss

```diff
@@ -1,6 +1,6 @@
 @import '../abstracts/_abstracts-dir';
 
-.footer-container {
+.footer {
```

Comment:
As you can see, we already have `grid` controlling the size, so no need for `respond-above` in here.